index.css

```
@@ -122,492 +121,494 @@ img {
-    background: var(--pink);
-    margin: var(--gutter-small) 0;
+  content: "";
+  display: block;
+  height: 2px;
+  width: 10rem;
+  background: var(--pink);
+  margin: var(--gutter-small) 0;
 }
 
 h3 {
-    font-size: var(--font-size-medium-1);
-    margin-bottom: var(--gutter-small);
+  font-size: var(--font-size-medium-1);
+  margin-bottom: var(--gutter-small);
 }
 
-@media(max-width: 500px) {
-    .heading-primary {
-        font-size: var(--font-size-large);
-    }
+@media (max-width: 500px) {
+  .heading-primary {
+    font-size: var(--font-size-large);
+  }
 
-    h2 {
-        font-size: var(--font-size-medium-1);
-    }
+  h2 {
+    font-size: var(--font-size-medium-1);
+  }
 
-    h3 {
-        font-size: var(--font-size-medium);
-    }
+  h3 {
+    font-size: var(--font-size-medium);
+  }
 }
 
 p {
-    margin: var(--gutter-small-1) 0;
+  margin: var(--gutter-small-1) 0;
 }
 
-@media(max-width: 900px) {
-    p { 
-        max-width: var(--container-small-width);
-    }
+@media (max-width: 900px) {
+  p {
+    max-width: var(--container-small-width);
+  }
 }
 
 /* --------------------------------------- */
 /* ----- Buttons and Links ----- */
 /* --------------------------------------- */
 
 a {
-    color: var(--white);
-    font-weight: 400;
-    font-family: 'Jost', sans-serif;
-    transition: all .2s;
-    outline: 2px solid transparent;
+  color: var(--white);
+  font-weight: 400;
+  font-family: "Jost", sans-serif;
+  transition: all 0.2s;
+  outline: 2px solid transparent;
 }
 
 ::-moz-focus-inner {
-    border: 0;
+  border: 0;
 }
 
 button:focus,
 a:focus {
-    outline: 2px solid var(--pink);
+  outline: 2px solid var(--pink);
 }
 
 body:not(.user-is-tabbing) button:focus,
 body:not(.user-is-tabbing) a:focus {
-    outline: none;
+  outline: none;
 }
 
 .link:hover {
-    color: var(--pink);
+  color: var(--pink);
 }
 
 .btn {
-    position: relative;
-    display: inline-block;
-    padding: 1rem 4.2rem;
-    text-decoration: none;
-    color: inherit;
-    border: 1px solid  var(--pink);
-    font-weight: 400;
+  position: relative;
+  display: inline-block;
+  padding: 1rem 4.2rem;
+  text-decoration: none;
+  color: #000;
+  font-weight: 400;
+  box-sizing: border-box;
 }
 
 .btn:focus {
-    outline: none;
+  outline: none;
 }
 
 .btn::after {
-    content: '';
-    display: block;
-    position: absolute;
-    top: 0;
-    left: 0;
-    right: 100%;
-    background: var(--white);
-    height: 100%;
-    z-index: -1;
-    transition: all 0.2s cubic-bezier(1, 0.68, 0.16, 0.9);
+  content: "";
+  display: block;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 100%;
+  background: var(--white);
+  height: 100%;
+  z-index: -1;
+  transition: all 0.2s cubic-bezier(1, 0.68, 0.16, 0.9);
 }
 
 .btn:hover::after,
 .btn:focus::after {
-    right: 0;
-    background: var(--pink);
+  right: 0;
+  background: var(--pink);
 }
 
 .btn--pink {
-    background: var(--pink);
-    transition: all 0.2s;
+  background: white;
```

Please revert this back to and maybe let's find a pink which has good contrast and update this here https://github.com/nisarhassan12/portfolio-template/blob/55fe0c5d0a95e856fa408578891ff1bc77cbf497/index.css#L38
```suggestion
  background: var(--pink);
```